web/client-ide/nginx/nginx.conf

```diff
@@ -0,0 +1,35 @@
+user  nginx;
+worker_processes  1;
+
+# This is important! because it loads several dav module with this.
+include /etc/nginx/modules-enabled/*.conf;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    #gzip  on;
+
+    dav_ext_lock_zone zone=notebooks:10m;
```

Comment:
What does this mean? Is 10m meaningful? May be worth a comment in code.

Reply:
It's the size of the shared zone for locks. I'll add a comment.